apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx

```diff
@@ -1,33 +0,0 @@
-import React, { useMemo } from 'react'
-import { Select, SelectProps } from 'antd'
-import get from 'lodash/get'
-import { OrganizationEmployee } from '../../../../schema'
-import { OrganizationEmployeeRole } from '../../utils/clientSchema'
-
-interface IEmployeeRoleSelectProps extends SelectProps<string> {
-    employee: OrganizationEmployee
-}
-
-export const EmployeeRoleSelect: React.FC<IEmployeeRoleSelectProps> = (props) => {
```

Comment:
why you delete it?

Reply:
Because there were duplication of compoennt